Index: .idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\r\n# Default ignored files\r\n/workspace.xml
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
--- a/.idea/.gitignore	(revision 3a50a07861814529e16612dbe5f9b65cac4b7ee6)
+++ b/.idea/.gitignore	(date 1613874600011)
@@ -1,3 +1,5 @@
-
 # Default ignored files
-/workspace.xml
\ No newline at end of file
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
Index: .idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"ProjectModuleManager\">\r\n    <modules>\r\n      <module fileurl=\"file://$PROJECT_DIR$/.idea/IS322Project2.iml\" filepath=\"$PROJECT_DIR$/.idea/IS322Project2.iml\" />\r\n    </modules>\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/modules.xml b/.idea/modules.xml
--- a/.idea/modules.xml	(revision 3a50a07861814529e16612dbe5f9b65cac4b7ee6)
+++ b/.idea/modules.xml	(date 1613874600162)
@@ -2,7 +2,7 @@
 <project version="4">
   <component name="ProjectModuleManager">
     <modules>
-      <module fileurl="file://$PROJECT_DIR$/.idea/IS322Project2.iml" filepath="$PROJECT_DIR$/.idea/IS322Project2.iml" />
+      <module fileurl="file://$PROJECT_DIR$/.idea/IS322Project.iml" filepath="$PROJECT_DIR$/.idea/IS322Project.iml" />
     </modules>
   </component>
 </project>
\ No newline at end of file
Index: .idea/IS322Project.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/IS322Project.iml b/.idea/IS322Project.iml
new file mode 100644
--- /dev/null	(date 1613874600085)
+++ b/.idea/IS322Project.iml	(date 1613874600085)
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$">
+      <excludeFolder url="file://$MODULE_DIR$/temp" />
+      <excludeFolder url="file://$MODULE_DIR$/.tmp" />
+      <excludeFolder url="file://$MODULE_DIR$/tmp" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
\ No newline at end of file
